File: TD/TD2/Code/1.cpp
/**
 * Calculer la somme des N première puissances de 2
 */
#include <cmath> // Contient la fonction pow()
#include <iostream>

int main()
{
  int a;
  int b;

  b = 0;

  std::cout << "Donne moi une valeur : " << std::flush;
  std::cin >> a;

  for (int i = 0; i < a; ++i)
    b += pow(2, i);

  std::cout << b << std::endl;

  return 0;
}

File: TD/TD2/Code/Python/1.py
#!/usr/bin/python
# Calculer la somme des N première puissances de 2
somme = 0

print('Donne moi une valeur : ', end='')
a = int(input())

for i in range(a):
    somme += pow(i, 2)  # Ou on peut utiliser 'somme += i ** 2'

print('La somme est', somme)
